block/local/adapter.go

```diff
@@ -93,6 +93,27 @@ func (l *Adapter) Remove(obj block.ObjectPointer) error {
 	return os.Remove(p)
 }
 
+func (l *Adapter) Copy(sourceObj, destinationObj block.ObjectPointer) error {
+	dest := l.getPath(destinationObj.Identifier)
+	destinationFile, err := os.Create(dest)
+	if err != nil {
+		return err
+	}
+	defer func() {
+		_ = destinationFile.Close()
```

Comment:
1. This ignores errors.
1. This leaves a bad (empty) file when the copy fails (or even if the source cannot be opened, because of create-before-open above).  It should instead delete it.